frontend/src/pages/projects/screens/spawner/environmentVariables/EnvTypeSelectField.tsx

```diff
@@ -18,50 +18,52 @@ const EnvTypeSelectField: React.FC<EnvTypeSelectFieldProps> = ({
   onUpdate,
   onRemove,
 }) => (
-  <Split data-testid="environment-variable-field">
-    <SplitItem isFilled>
-      <Stack hasGutter>
-        <StackItem data-testid="environment-variable-type-select">
-          <SimpleSelect
-            popperProps={{ appendTo: getDashboardMainContainer() }}
-            isFullWidth
-            toggleLabel={envVariable.type || 'Select environment variable type'}
-            options={Object.values(EnvironmentVariableType).map((type) => ({
-              key: type,
-              label: type,
-            }))}
-            onChange={(value) => {
-              const enumValue = asEnumMember(value, EnvironmentVariableType);
-              if (enumValue !== null) {
-                onUpdate({
-                  type: enumValue,
-                });
-              }
-            }}
-          />
-        </StackItem>
-        {envVariable.type && (
-          <StackItem>
-            <IndentSection>
-              <EnvTypeSwitch
-                env={envVariable}
-                onUpdate={(envValue) => onUpdate({ ...envVariable, values: envValue })}
-              />
-            </IndentSection>
+  <FormGroup isRequired label="Variable type">
+    <Split data-testid="environment-variable-field">
+      <SplitItem isFilled>
+        <Stack hasGutter>
+          <StackItem data-testid="environment-variable-type-select">
+            <SimpleSelect
+              popperProps={{ appendTo: getDashboardMainContainer() }}
```

Comment:
```suggestion
  <FormGroup isRequired label="Variable type" fieldId="environment-variable-type-select">
    <Split data-testid="environment-variable-field">
      <SplitItem isFilled>
        <Stack hasGutter>
          <StackItem data-testid="environment-variable-type-select">
            <SimpleSelect
              toggleProps={{ id: 'environment-variable-type-select' }}
              popperProps={{ appendTo: getDashboardMainContainer() }}
```

So I noticed this accessibility error when working on my own PR when adding a label to a SimpleSelect. The FormGroup label doesn't point to anything. But you can give the SimpleSelect a toggle props with the id and it will fix it
![image](https://github.com/user-attachments/assets/b075ed37-30a6-4a74-b99f-135ced19b642)